src/types/contracts.ts

```diff
@@ -12,6 +12,7 @@ export interface IContract {
   notes: string;
   contractType: string;
   paymentSettlement: IPaymentSettlement[];
+  endReasonID: string;
```

Comment:
Please change to `endReasonId `to keep to the pattern.

Reply:
Changed.